battenberg_gitlab/apply.py

```diff
@@ -0,0 +1,165 @@
+import os
+import logging
+import json
+from typing import List
+from functools import reduce
+from gitlab import Gitlab
+from pygit2 import Repository
+from battenberg import Battenberg
+from battenberg.errors import MergeConflictException
+from battenberg_gitlab.search import get_projects
+from battenberg_gitlab.utils import clone_or_discover_repo, ensure_workspace
+
+
+logger = logging.getLogger(__name__)
+
+
+# TODO Remove these once we've we've backfilled .cookiecutter.json
+DEFAULTS = {
+    "_template": "git@gitlab.zgtools.net:zillow-offers/zoml/ml-infrastructure/archetype.py-ml.git",
+    "author": "Zillow Offers ML",
+    "aws_profile_name": "zillow-offers-machine-learning-developer-role",
+    "email": "zoml@zillowgroup.com",
+    "generate_example": "False",
+    "model_holder_type": "ModelHolder",
+    # "module_name": "example_project",
+    # "project_description": "Provides an example use case of the ML archetype",
+    # "project_name": "example-project",
+    "project_version": "v1",
+    "team_short_name": "zoml",
+    # "variant_name": "linear_model"
+}
+TEMPLATES = {
```

Comment:
I really don't love these being public on GitHub

Reply:
Cool moved to a `templates.json` file and ignored that one. Also rebased the commit to remove it from Github.